consensus/istanbul/backend/engine_test.go

```diff
@@ -169,14 +169,14 @@ func TestSealStopChannel(t *testing.T) {
 		stop <- struct{}{}
 		eventSub.Unsubscribe()
 	}
-	go eventLoop()
 	resultCh := make(chan *types.Block, 10)
 	go func() {
 		err := engine.Seal(chain, block, resultCh, stop)
 		if err != nil {
 			t.Errorf("error mismatch: have %v, want nil", err)
 		}
 	}()
+	go eventLoop()
```

Comment:
this change seems unrelated to title. what was it about?

Reply:
Yes, the change is unrelated to the PR. When I created the PR, this test failed. On looking closely, the test was failing like 1 in 250 - 300 runs. Moving the call to `eventLoop()` ensures that `stop` channel is not triggered before `Seal()` is called